oci-resources.py

```diff
@@ -27,9 +27,9 @@
 # Output formats for readable, columns style output and csv files
 field_names = [
 	'Tenancy', 'Region', 'Compartment', 'Type', 'Name', 'State', 'DB',
-	'Shape', 'OCPU', 'StorageGB', 'BYOLstatus', 'Created']
+	'Shape', 'OCPU', 'StorageGB', 'BYOLstatus', 'VolAttached', 'Created' ]
 print_format = '{Tenancy:24s} {Region:9s} {Compartment:54s} {Type:20s} {Name:54.54s} {State:18s} {DB:4s} ' \
-			   '{Shape:20s} {OCPU:>4s} {StorageGB:>9s} {BYOLstatus:10s} {Created:32s}'
+		'{Shape:20s} {OCPU:>4s} {StorageGB:>9s} {BYOLstatus:10s} {VolAttached:32s} {Created:32s} '
```

Comment:
field length still 32 - it's too long. Only needs to be 12 to contain "Not Attached"